lib/elementWrapper/checkBoxWrapper.js

```diff
@@ -0,0 +1,32 @@
+const CheckBox = require('../elements/checkBox');
+const ElementWrapper = require('./elementWrapper');
+class CheckBoxWrapper extends ElementWrapper {
+  async isChecked() {
+    const elems = await this.elements();
```

Comment:
@sriv this will intoduce `firstElement` API on all the ElementWrappers.
If this is not what we are looking for, we can extract this peace of code into a helper.

Reply:
I thought we want ElementWrappers to return a list of matches and delegate to the first match when an element's method is invoked?

Reply:
Yes, that is the expected behaviour for `ElementWrappers`